backend/post/views.py

```diff
@@ -1,18 +1,19 @@
 from django.shortcuts import render
 from rest_framework import viewsets
 from rest_framework.response import Response
+from rest_framework.exceptions import ValidationError
 from .serializers import PostSerializer
 from .models import Post
 
 
 # Create your views here.
 class PostView(viewsets.ModelViewSet):
-    queryset = Post.objects.order_by('-published_at')
+    queryset = Post.objects
     serializer_class = PostSerializer
     lookup_field = 'slug'
 
 class PostDetailView(viewsets.ModelViewSet):
-    queryset = Post.objects.order_by('-published_at')
+    queryset = Post.objects
```

Comment:
Same here:
queryset = Post.objects.all() 
Please correct me if I'm wrong hehe thank you!

Reply:
yes, I forgot I already changed it. Thanks

Reply:
Thank you. Looks good to me.